mllib/src/test/scala/org/apache/spark/ml/classification/MultilayerPerceptronClassifierSuite.scala

```diff
@@ -229,4 +229,17 @@ class MultilayerPerceptronClassifierSuite extends MLTest with DefaultReadWriteTe
         assert(expected.weights === actual.weights)
       }
   }
+
+  test("Load MultilayerPerceptronClassificationModel prior to Spark 3.0") {
```

Comment:
Just a general question, we also have other algorithms that modified the load/save method (like NaiveBayes), do we need to add testsuites for them like this?

Reply:
I am ok either way.

Reply:
so add similar test in ```NaiveBayes``` (and other algorithms that modified the load/save method) too?

Reply:
Maybe needed, but in other PRs.